spring-cloud-gcp-examples/pom.xml

```diff
@@ -21,5 +21,6 @@
     <modules>
 		<module>spring-cloud-gcp-config-example</module>
 		<module>spring-cloud-gcp-trace-example</module>
+		<module>spring-cloud-gcp-spring-integration-pubsub-example</module>
```

Comment:
Double `spring`. How about `spring-integration-cloud-gcp-pubsub-example` ? 
Or just without the second `spring`, or just `pubsub`, without `spring-integration`